src/Assetic/Filter/JSqueezeFilter.php

```diff
@@ -23,7 +23,20 @@ class JSqueezeFilter implements FilterInterface
 {
     private $singleLine = true;
     private $keepImportantComments = true;
-    private $specialVarRx = \JSqueeze::SPECIAL_VAR_RX;
+    private $classname;
+    private $specialVarRx;
+
+    function __construct() {
+        // JSqueeze is namespaced since 2.x, this works with both 1.x and 2.x
+        // Upstream default for special var is false in 2.x
+        if (class_exists('\\Patchwork\\JSqueeze')) {
+            $this->classname = '\\Patchwork\\JSqueeze';
+            $this->specialVarRx = false;
```

Comment:
false by default in 1 and 2 would be fine for me. As you noticed, the option allows increasing the compression ratio but lets have a safer default